pcsx2/CDVD/CDVD.cpp

```diff
@@ -2247,21 +2249,115 @@ static void cdvdWrite16(u8 rt) // SCOMMAND
 						break;
 
 					case 0xFD: // _sceCdReadRenewalDate (1:6) BCD
-						SetSCMDResultSize(6);
-						cdvd.SCMDResult[0] = 0;
-						cdvd.SCMDResult[1] = 0x04; //year
-						cdvd.SCMDResult[2] = 0x12; //month
-						cdvd.SCMDResult[3] = 0x10; //day
-						cdvd.SCMDResult[4] = 0x01; //hour
-						cdvd.SCMDResult[5] = 0x30; //min
+						switch (temp_mechaver[1])
+						{
+							case 0x01:
+							case 0x02:
+							case 0x03:
+								// Mechacon version prior to v5 doesnt support SCMD 03,FD command
+								SetSCMDResultSize(1);
+								cdvd.SCMDResult[0] = 0x80;
+								break;
+
+							case 0x05:
+								SetSCMDResultSize(6);
+								switch (temp_mechaver[2])
+								{
+									case 0x00:
+									case 0x01:
+										memcpy(cdvd.SCMDResult, &MRenewalDate[0], 6);
+										break;
+									case 0x02:
+									case 0x03:
+										memcpy(cdvd.SCMDResult, &MRenewalDate[1], 6);
+										break;
+									case 0x04:
+									case 0x05:
+										memcpy(cdvd.SCMDResult, &MRenewalDate[2], 6);
+										break;
+									case 0x06:
+									case 0x07:
+										if (temp_mechaver[0] != 0x07)
+											memcpy(cdvd.SCMDResult, &MRenewalDate[3], 6);
+										else
+											memcpy(cdvd.SCMDResult, &MRenewalDate[4], 6);
+										break;
+									case 0x0A:
+									case 0x0B:
+										memcpy(cdvd.SCMDResult, &MRenewalDate[5], 6);
+										break;
+									case 0x0C:
+									case 0x0D:
+										memcpy(cdvd.SCMDResult, &MRenewalDate[6], 6);
+										break;
+									case 0x0E:
+									case 0x0F:
+										memcpy(cdvd.SCMDResult, &MRenewalDate[7], 6);
+										break;
+									default:
+										SetSCMDResultSize(1);
+										cdvd.SCMDResult[0] = 0x80;
+										break;
+								}
+								break;
+
+							case 0x06:
+								SetSCMDResultSize(6);
+								switch (temp_mechaver[2])
+								{
+									case 0x00:
+									case 0x01:
+										memcpy(cdvd.SCMDResult, &MRenewalDate[8], 6);
+										break;
+									case 0x02:
+									case 0x03:
+										memcpy(cdvd.SCMDResult, &MRenewalDate[9], 6);
+										break;
+									case 0x04:
+									case 0x05:
+										memcpy(cdvd.SCMDResult, &MRenewalDate[10], 6);
+										break;
+									case 0x06:
+									case 0x07:
+										memcpy(cdvd.SCMDResult, &MRenewalDate[11], 6);
+										break;
+									case 0x0A:
+									case 0x0B:
+										memcpy(cdvd.SCMDResult, &MRenewalDate[12], 6);
+										break;
+									case 0x0C:
+									case 0x0D:
+										memcpy(cdvd.SCMDResult, &MRenewalDate[13], 6);
+										break;
+									default:
+										SetSCMDResultSize(1);
+										cdvd.SCMDResult[0] = 0x80;
+										break;
+								}
+								break;
```

Comment:
as above but +8

Reply:
5.8, 5.9, 6.8, 6.9 does not exist. I  added 0x80 response into enumartor